buildenv/jenkins/openjdk_tests

```diff
@@ -170,30 +170,31 @@ timestamps{
                 LABEL = params.LABEL
             } else {
                 LABEL = PLATFORM_MAP[params.PLATFORM]["LABEL"]
-                    if (params.BUILD_LIST.contains("perf")) {
-                        def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
-                        if (areNodesWithLabelOnline(perfLabel)) {
+                if (params.BUILD_LIST.contains("perf")) {
+                    def perfLabel = LABEL.minus("ci.role.test&&").concat("&&ci.role.perf")
+                    if (areNodesWithLabelOnline(perfLabel)) {
                         LABEL = perfLabel
-                        }
                     }
                 }
+                if (params.TEST_FLAG == "FIPS") {
+                    LABEL = LABEL.minus("ci.role.test&&").concat("&&ci.role.test.fips")
+                }
+                // Temporarily exclude ubuntu 22 machines for criu sanity.external pipeline
+                if (JOB_NAME.contains ("sanity.external") && JOB_NAME.contains("_criu")) {
+                    LABEL += "&&!sw.os.ubuntu.22"
+                }
 
-            if (params.TEST_FLAG == "FIPS") {
-                LABEL = LABEL.minus("ci.role.test&&").concat("&&ci.role.test.fips")
-                env.EXTRA_OPTIONS += " -Dsemeru.fips=true"
-            }
-
-            // Temporarily exclude ubuntu 22 machines for criu sanity.external pipeline
-            if (JOB_NAME.contains ("sanity.external") && JOB_NAME.contains("_criu")) {
-                LABEL += "&&!sw.os.ubuntu.22"
-            }
+                if (params.DOCKER_REQUIRED) {
+                    LABEL += "&&sw.tool.docker"
+                }
 
-            if (params.DOCKER_REQUIRED) {
-                LABEL += "&&sw.tool.docker"
+                if (params.LABEL_ADDITION) {
+                    LABEL += "&&${params.LABEL_ADDITION}"
+                }
```

Comment:
Will this change the behavior? If `params.LABEL` is not provided by the users, `params.LABEL_ADDITION` will not work.

Reply:
If user would like to specify the agent or machine I think it should be user's responsibility to first  figure the correct the agent which satisfy all the label requirements for the jenkins job.

Reply:
This `user` can be our test pipeline. In the Semeru test pipeline, it sets `params.LABEL_ADDITION` in some special cases and leave `params.LABEL` empty. This change will break our pipeline.